tests/ui/artificial-block.rs

```diff
@@ -1,5 +1,30 @@
+//! Check that we don't get compile errors on unreachable code after the `{ ret 3; }` artificial
+//! block below. This test is run-pass to also exercise the codegen, but it might be possible to
+//! reduce to build-pass or even check-pass.
+//!
+//! This test was introduced as part of commit `a833f152baa17460e8414355e832d30d5161f8e8` which
+//! removes an "artificial block". See also commit `8d381823e2aa1524eabeb3219d7dc1d5007e6096` for
```

Comment:
You are 100% correct. I somehow conjured a *completely* unrelated **full** commit hash into this comment, the correct hash is actually 3d738e9e0634a4cd6239d1317bd7dad53be68dc8, thankfully I reproduced the commit message otherwise I would not have been able to find it.

Reply:
Fixed